process.yml

```diff
@@ -0,0 +1,19 @@
+apps:
+    - name: api
+      port: 3002
+      script: ./dist/server/server.js
+      instances: 1
+      watch: true
+      exec_mode: cluster
+      env:
+        NODE_ENV: production
```

Comment:
Setz auch noch `CACHE_PATH` auf `/home/concourse/build-result/cache` o. ä., in der Hoffnung, dass der beim rsync eines neuen Stands gelöscht wird

Reply:
done